tests/requirements.txt

```diff
@@ -1,6 +1,6 @@
 coverage==4.5.1
 docutils==0.14
-flake8==3.5.0
+git+https://github.com/PyCQA/flake8 # TODO: Remove after release https://gitlab.com/pycqa/flake8/merge_requests/230
```

Comment:
probably better to revert `pycodestyle==2.3.1` or whatever rather than use an unreleased version of flake8?

Reply:
Done